math.c

```diff
@@ -170,6 +170,12 @@ math_tan(VALUE unused_obj, VALUE x)
     return DBL2NUM(tan(Get_Double(x)));
 }
 
+#define math_arc(num, msg, func) \
+    double d; \
+    d = Get_Double((num)); \
+    domain_check_range(d, -1.0, 1.0, msg); \
```

Comment:
You can omit to repeat `func` and `msg`.

```suggestion
#define math_arc(num, func) \
    double d; \
    d = Get_Double((num)); \
    domain_check_range(d, -1.0, 1.0, #func); \
```

Reply:
Thanks!
Fixed it!